client/common/cmdline.c

```diff
@@ -589,7 +589,7 @@ static char** freerdp_command_line_parse_comma_separated_values_ex(const char* n
 
 			if (p)
 			{
-				p[0] = name;
+				p[0] = (char*)name;
```

Comment:
This is something that needs to be fixed, so don't cast the warning away

Reply:
A few more lines down there is already such an assignment with the cast. In both cmdline.c files.
I tested both versions and both work.

Reply:
@hendwolt yes, there are still many such things to be sorted out :/ They, of course, work but silence the compiler (which makes it easy to forget to fix)